index.bs

```diff
@@ -2690,6 +2733,31 @@ non-negative integer |defaultBudget|, and an [=aggregatable debug reporting conf
     with |value|, |budget|, |supportedTypes|, and |defaultConfig|.
 1. Return the [=tuple=] (|budget|, |config|).
 
+To <dfn>parse max event states</dfn> from a [=map=] |map| and
+a possibly null 32-bit positive integer |attributionScopeLimit|:
+1. If |attributionScopeLimit| is null, return [=default max event states=].
```

Comment:
To be consistent with attribution scopes parsing, I think it's better to remove this step, and instead after parsing check attribution scope limit against the parsed value and return an error if the attribution scope limit is not null and the parsed value is not the default.